### DISCUSSION TITLE
Export to SVG

### DISCUSSION THREAD

**Discussion by piotrtrkostrzewski94** [Orama-Interactive/Pixelorama#968] [Ideas]
If it is possible, please add an export option to SVG in Pixelorama.
I know that there is a plugin for Godot: https://godotengine.org/asset-library/asset/1544 but it has not been developed for a year, so it would probably require adaptation to the current version of Pixelorama, if it can be implemented directly in the code.
Ps.We wrote about it on Discord, but I can follow any progress here.  
Have a good night.

**Comment by TheLsbt** [comment]
I remember I found this a while ago [Blocky](https://github.com/tinychameleon/blocky) this a bit ago. It's written in go but should be easy enough to translate to GDScript
